### ISSUE TITLE
Screen reader does not announce error message

### ISSUE THREAD

**Issue by Saurabh19Aug** [microsoft/vscode#107020]
**(This was initially reported against the monaco editor, so it might not reproduce in newer versions of vscode)**

Simplified steps:
* turn on Narrator on Windows
* open https://vscode-web-test-playground.azurewebsites.net/?enter=true
* press Shift+F8 
* Narrator does not announce error message

![MAS4 2 1_error msg not reading](https://user-images.githubusercontent.com/59047554/89535928-be303f80-d814-11ea-9908-3d586c0aafc9.PNG)

---

<details><summary>ORIGINAL ISSUE BODY</summary>

**User Experience:**

Users who depends on screen reader would not able get error/warning message information if screen reader does not announce about error message, screen reader should announce the error/warning message as soon as it appears on the screen.

 

Note: User credentials should NOT be included in the bug.

**Prerequisite:** Please add or set required properties from customize your setting) on the control like Aria properties, state of the control(Valid, invalid, value) etc.

 

**Repro Steps:**

1. Open https://aka.ms/portalfx/playground and valid credentials.

2. Navigate to the control playground and select it.

3. Navigate to the "DiffEditor (TypeScriptEditor)" an select it.

5. Navigate to the control and control properties appearing on the "Sample" Pane.

6 .Enable screen reader and navigate to Editor pane and press (Shift + F8) key find out errors.

7.Verify the issue.

 

**Actual Result:**

Screen reader does not announce error message which is displayed after pressing press (Shift + F8) key to find out errors in editor pane.

 

**Expected Result:**

Screen reader should announce the error message which is displayed after pressing press (Shift + F8) key to find out errors in editor pane.

 

**Note:**

Similar issue exist throughout the application.

This issue does not exist with Firefox + NVDA and Chrome + JAWS

**Test Environment:**
OS: Version 2004(OS Build 19041.329)
Edge Chromium: Version 85.0.538.0 (Official build) dev (64-bit)
Screen Reader: Narrator
#URL: https://aka.ms/portalfx/playground 
![MAS4 2 1_error msg not reading](https://user-images.githubusercontent.com/59047554/89535928-be303f80-d814-11ea-9908-3d586c0aafc9.PNG)

**MAS Reference:**

MAS 4.2.1 - Object Information (33A)

        

**Reference Links**

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

Bug copied from Control Playground : <https://msazure.visualstudio.com/One/_workitems/edit/7499425> 

</details>

**Comment by sandy081** [comment]
Voice Over in Mac announces the error 

<img width="832" alt="image" src="https://user-images.githubusercontent.com/10746682/93878476-6bceb380-fcda-11ea-8b60-e2e10f5a08ad.png">

@isidorn any clue? what's difference with windows narrator?

**Comment by isidorn** [comment]
We do not really care about windows narrator since none of the users actually use it. We support NVDA on Windows.

@saurabh19aug can you reproduce the issue with NVDA?
